testing/src/main/java/io/stargate/it/http/RestApiv2Test.java

```diff
@@ -107,6 +109,7 @@ public MapGetResponseWrapper() {
   @BeforeEach
   public void setup(TestInfo testInfo, StargateConnectionInfo cluster) throws IOException {
     host = "http://" + cluster.seedAddress();
+    restUrlBase = host + ":8082";
```

Comment:
Was not 100% sure if "cql port" of cluster would work here or not? 8082 appears to be hardcoded in code all over tho.